client/src/components/Attachment/Attachment.css

```diff
@@ -0,0 +1,171 @@
+/******** FILE UPLOAD AREA ********/
+.FileUploadContainer {
```

Comment:
Naming of CSS classes seems to be inconsistent; I see PascalCase, camelCase and lower-case-with-dashes, all intermixed. Please try to be consistent, and most customary is ower-case-with-dashes.